<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1590</width>
    <height>933</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="button_widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>931</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QPushButton" name="Use_camera">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Use Camera</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Upload_video">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Upload Video</string>
     </property>
    </widget>
    <widget class="QPushButton" name="playBtn">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>10</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>START</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopBtn">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>10</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>STOP</string>
     </property>
    </widget>
   </widget>
   <widget class="QVideoWidget" name="videowidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>931</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="video_progress" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>931</width>
      <height>41</height>
     </rect>
    </property>
    <widget class="QLabel" name="position_label">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>10</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>741</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>931</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="athelete_data" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>401</width>
      <height>241</height>
     </rect>
    </property>
    <widget class="QLabel" name="name">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Athlete Name :</string>
     </property>
    </widget>
    <widget class="QLabel" name="height">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Height :</string>
     </property>
    </widget>
    <widget class="QLabel" name="BMI">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>BMI:</string>
     </property>
    </widget>
    <widget class="QLabel" name="date_and_time">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Date and Time:</string>
     </property>
    </widget>
    <widget class="QLabel" name="ip_height">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>.../cm</string>
     </property>
    </widget>
    <widget class="QLabel" name="weifht">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Weight:</string>
     </property>
    </widget>
    <widget class="QLabel" name="ip_weight">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>80</y>
       <width>59</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>.../kg</string>
     </property>
    </widget>
    <widget class="QLabel" name="ip_BMI">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>... kg/m^2</string>
     </property>
    </widget>
    <widget class="QLabel" name="IP_DATE">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>date...</string>
     </property>
    </widget>
    <widget class="QLabel" name="age">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Age:</string>
     </property>
    </widget>
    <widget class="QLabel" name="ip_age">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>59</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>.age</string>
     </property>
    </widget>
    <widget class="QLabel" name="ip_ame">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>60</y>
       <width>59</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>...name</string>
     </property>
    </widget>
   </widget>
   <widget class="PlotWidget" name="sensors_pg" native="true">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>630</y>
      <width>501</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>560</y>
      <width>20</width>
      <height>301</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="Waveform" native="true">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>0</y>
      <width>641</width>
      <height>861</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Parameters</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>151</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>力道 (Strength)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>300</y>
       <width>411</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>轉速 (TWIST &amp; TURN ROTATION SPEED)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>580</y>
       <width>251</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>滯空時間 (HAND OFF TIME)</string>
     </property>
    </widget>
    <widget class="PlotWidget" name="head_height_pg" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>591</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="PlotWidget" name="hand_off_pg" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>630</y>
       <width>591</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="PlotWidget" name="twist_pg" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>591</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>20</y>
      <width>20</width>
      <height>871</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>570</y>
      <width>321</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Vibration seneors measurements</string>
    </property>
   </widget>
   <widget class="QLabel" name="athelete_title">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>660</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>                 Athllete and Video Information</string>
    </property>
    <property name="margin">
     <number>-2</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1590</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionMivos"/>
    <addaction name="actionMotion_Analysis"/>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Exit</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuWindows"/>
   <addaction name="menuExit"/>
  </widget>
  <action name="actionMivos">
   <property name="text">
    <string>Mivos</string>
   </property>
  </action>
  <action name="actionMotion_Analysis">
   <property name="text">
    <string>Motion Analysis</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
